llama_hub/llama_packs/panel_chatbot/base.py

```diff
@@ -20,7 +22,7 @@ def run(self, *args: Any, **kwargs: Any) -> Any:
         """Run the pipeline."""
         for variable in ENVIRONMENT_VARIABLES:
             if variable not in os.environ:
-                raise ValueError(f"{variable} is not set")
+                logging.warn("%s environment variable is not set", variable)
```

Comment:
I would prefer to raise an exception here. But the pytests will break because the environment variables are not set during testing.